docs/twig-components/method/none.md

```diff
@@ -1,6 +1,12 @@
 # none
 
-`none` is a Twig test to ...
+`none` is a Twig test to check if the variable is null.
 
+```twig
+{{ var is null }}
+```
 
-Source: 
\ No newline at end of file
+Note
```

Comment:
Is a colon (:) missing after `Note`? Or maybe it all just a sentence that should be in one line?